### PR TITLE
Vy res 10

### PR DESCRIPTION
**Pull request by joshianuj**
Front end basic design setup

### CODE REVIEW

File: frontend/gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        // run it once the first time buildScript is called
+        return rebundle();
+    }
+
+    gulp.task('scripts', function() {
+        return buildScript(false); // this will run once because we set watch to false
+    });
+
+// run 'scripts' task first, then watch for future changes
+    gulp.task('default', ['styles','scripts','browser-sync', 'image', 'fonts'], function() {
```
Comment (bishalshr) [inline frontend/gulpfile.js]: you can use images instead of image

File: frontend/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <link rel="stylesheet" href="dist/css/bundle.css">
+
+</head>
+
+<body>
+<div id="vyaguta-body">
+
+</div>
+
+
```
Comment (bishalshr) [inline frontend/index.html]: you can remove unwanted spaces

File: frontend/src/app.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+;(function(){
```
Comment (bishalshr) [inline frontend/src/app.js]: use 'use strict'

File: frontend/src/app.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    var $ = jQuery = require("jquery");
+    var React = require("react");
```
Comment (bishalshr) [inline frontend/src/app.js]: use single quote

File: frontend/src/routes/routes.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+;(function(){
```
Comment (pratishshr) [inline frontend/src/routes/routes.js]: 'use strict'

**Comment by joshianuj** [comment]
:ok:

**Pull request merged**